Formula/vgmstream.rb

```diff
@@ -1,15 +1,27 @@
 class Vgmstream < Formula
   desc "Library for playing streamed audio formats from video games"
   homepage "https://vgmstream.org"
-  url "https://github.com/vgmstream/vgmstream.git",
-      tag:      "r1667",
-      revision: "6b84f258e4238edd627e24ec8460a7040613d054"
-  version "r1667"
+  version "r1702"
   license "ISC"
-  revision 1
   version_scheme 1
   head "https://github.com/vgmstream/vgmstream.git", branch: "master"
 
+  stable do
+    url "https://github.com/vgmstream/vgmstream.git",
+        tag:      "r1702",
+        revision: "a76ac04dcd41f1926a721e626657397bd9656a74"
+
+    # patches for fixing macOS build error, remove them at next release
+    patch do
+      url "https://github.com/vgmstream/vgmstream/commit/04b5a1f098f348c4c438ada85ee86f5a9abce2ff.patch?full_index=1"
+      sha256 "37d5d9567435eda96bf4e51b5abbac1677391e08daf4e50b1d7edc6996772919"
+    end
+    patch do
+      url "https://github.com/vgmstream/vgmstream/commit/f9230158d953dfb0fb14a2a57052a8460a5d43dd.patch?full_index=1"
+      sha256 "10ff07d7c6f48275af235d720f279ef62d444c88cde78f2fecaf065e120d134d"
+    end
```

Comment:
We actually set `CC` and `CXX` in the environment, so this patch probably doesn't do anything. It doesn't hurt, though, so there's no need to remove it.